libs/ui/search/src/lib/organisation-preview/organisation-preview.component.ts

```diff
@@ -0,0 +1,14 @@
+import { ChangeDetectionStrategy, Component, Input } from '@angular/core'
+
+@Component({
+  selector: 'gn-ui-organisation-preview',
+  templateUrl: './organisation-preview.component.html',
+  styleUrls: ['./organisation-preview.component.css'],
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class OrganisationPreviewComponent {
+  @Input() title: string
+  @Input() description: string
+  @Input() logo: string
+  @Input() nRecords: number
```

Comment:
Maybe create model and rename `nRecords` as suggested earlier ?

Reply:
Yes but you create an Organisation model in your PR no ?